rpcs3/Emu/RSX/Overlays/overlay_controls.h

```diff
@@ -1095,10 +1095,19 @@ namespace rsx
 			u16 caret_position = 0;
 			u16 vertical_scroll_offset = 0;
 
-			bool m_reset_caret_pulse = 0;
+			bool m_reset_caret_pulse = false;
+			bool password_mode = false;
+
+			std::u32string value;
```

Comment:
Why introduce a duplicate variable when the base class has a text variable for this very purpose?

Reply:
Nvm, I missed the password mode inclusion.

Reply:
That. and also there would be no way to really discriminate if a text is a placeholder without some extra logic anyway.

Reply:
Another thing worth mentioning is that the old code allowed to move the cursor while the placeholder was shown.
This was obviously wrong and is now fixed, since the caret logic works with the value member instead of the text member.